utils/releaser/src/ReleaseWorker/Release/TagPhpPlatformReleaseWorker.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\Releaser\ReleaseWorker\Release;
+
+use PharIo\Version\Version;
+use Shopsys\Releaser\FileManipulator\DockerfileVersionFileManipulator;
+use Shopsys\Releaser\ReleaseWorker\AbstractShopsysReleaseWorker;
+use Shopsys\Releaser\ReleaseWorker\Message;
+use Shopsys\Releaser\Stage;
+
+final class TagPhpPlatformReleaseWorker extends AbstractShopsysReleaseWorker
+{
+    /**
+     * @param \Shopsys\Releaser\FileManipulator\DockerfileVersionFileManipulator $dockerfileVersionFileManipulator
+     */
+    public function __construct(
+        private readonly DockerfileVersionFileManipulator $dockerfileVersionFileManipulator,
+    ) {
+    }
+
+    /**
+     * @param \PharIo\Version\Version $version
+     * @param string $initialBranchName
+     * @return string
+     */
+    public function getDescription(
+        Version $version,
+        string $initialBranchName = AbstractShopsysReleaseWorker::MAIN_BRANCH_NAME,
+    ): string {
+        return 'Tag php-platform with the same version as the release and replace FROM in Dockerfile';
+    }
+
+    /**
+     * @param \PharIo\Version\Version $version
+     * @param string $initialBranchName
+     */
+    public function work(
+        Version $version,
+        string $initialBranchName = AbstractShopsysReleaseWorker::MAIN_BRANCH_NAME,
+    ): void {
+        $tempDirectory = trim($this->processRunner->run('mktemp -d -t shopsys-release-XXXX'));
+        $packageName = 'php-platform';
+        $versionString = $version->getOriginalString();
+
+        $this->symfonyStyle->note(sprintf('Cloning shopsys/%s. This can take a while.', $packageName));
+        $this->processRunner->run(
+            sprintf('cd %s && git clone https://github.com/shopsys/%s.git', $tempDirectory, $packageName),
+        );
+
+        $this->processRunner->run(
+            sprintf(
+                'cd %s/%s && git checkout %s && git tag %s',
+                $tempDirectory,
+                $packageName,
+                $initialBranchName,
+                $versionString,
+            ),
+        );
```

Comment:
Why is this separated here and not part of monorepo-split?

Reply:
This tagging has to be done prior to the monorepo-split (the one that splits tags) to ensure the `FROM` instruction can be safely updated and committed and the application can be built after the split.